misk-aws/src/test/kotlin/misk/jobqueue/sqs/DockerSqs.kt

```diff
@@ -20,13 +20,13 @@ internal class DockerSqs {
   // NB(mmihic): Because the client port is embedded directly into the queue URLs, we have to use
   // the same external port as we do for the internal port
   private val clientPort = ExposedPort.tcp(CLIENT_PORT)
-  private val composer = Composer("e-sqs", Container {
+  private val composer = Composer("e-sqs", Container({
     withImage("pafortin/goaws")
         .withName("sqs")
         .withCmd(listOf("goaws"))
         .withExposedPorts(clientPort)
         .withPortBindings(Ports().apply { bind(clientPort, Ports.Binding.bindPort(CLIENT_PORT)) })
-  })
+  }, {}))
```

Comment:
maybe don't require this arg? or create multiple constructors?

Reply:
I'll do the multiple constructors. Couldn't find a way to not require with a previous lambda there as an arg.

Reply:
there is also `ResourceLoader.SYSETM`, but i'd actually like to kill that since it makes `InjectorTest`s impossible. would be _fine_ in tests though.